spec/README.md

```diff
@@ -24,6 +24,36 @@ The source of this diagram can be found [here](../images/envelope_relationships.
 The [validation model] provides pseudocode showing how these layers fit
 together. See the [documentation] for more background and examples.
 
+## Tagged Releases
+
+The latest [tagged release] version matches the [SemVer](https://semver.org)
+MAJOR.MINOR version of the Attestation Framework spec.
+
+Backwards-compatible semantic updates to the spec (except predicates) are
+indicated through new tagged MINOR version releases.
+We use new tagged PATCH version releases to indicate updates to predicate
+specifications and/or backwards-compatible changes to the language bindings.
+
+### Examples
+
+-   Attestation Framework tagged release v1.0.2 (PATCH version) incorporates
+    refinements to the predicate specification process, a new predicate type,
```

Comment:
IMO it is a bit confusing for semantic versioning of the spec to _not_ apply to predicate changes (which I agree with) while also saying a spec version ships with a new predicate. Maybe the version of the spec should entirely omit predicates and it's just a matter of convenience that predicates live alongside the spec?

This does get a bit confusing with versioning of the code gen libraries though, as you highlight here.

Reply:
I see what you mean. The intent here isn't to say that none of the things under the PATCH version category really affect the core spec. Maybe we can take out the mention of predicate types altogether and say that PATCH versions are only for doc edits and language binding changes?

Reply:
I think that might be better!